### PR TITLE
SVGGenerator for exporting charts in the server side.

### LINKED ISSUE #1111: Add API to generate SVG for chart instance
In Charts for Vaadin 8 SVGGenerator allowed obtaining the String SVG for a Chart.
SVGGenerator was based on PhantomJS which has been deprecated for 3 years already and we should evaluate alternatives.
Possible alternatives are local headless chrome or local Highcharts export server
Related to vaadin/vaadin-charts-flow#264

Research timebox: 2 days

### PR DESCRIPTION
**Pull request by sosa-vaadin**
Generate an SVG string of a chart using a Configuration instance with the chart's data.

Related: #1111 

Example usage:
```java

Configuration chartConfiguration = createChartConfiguration();
String svg;
// using a try-with-resources block is recommended
try (SVGGenerator generator = new SVGGenerator()) {
    svg = generator.generate(chartConfiguration);
}
// svg now holds the chart as an SVG string
```

### CODE REVIEW

File: vaadin-charts-flow-parent/vaadin-charts-flow-svg-generator/src/main/java/com/vaadin/flow/component/charts/export/NodeRunner.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package com.vaadin.flow.component.charts.export;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.vaadin.flow.server.frontend.FrontendTools;
+import com.vaadin.flow.server.frontend.FrontendUtils;
+
+public class NodeRunner {
```
Comment (knoobie) [inline vaadin-charts-flow-parent/vaadin-charts-flow-svg-generator/src/main/java/com/vaadin/flow/component/charts/export/NodeRunner.java:10]: I saw the PR and description and was like AWESOME! I saw the node dependency on runtime and was like.. okay.. that's unexpected.. this should be mentioned as requirement at least.

Comment (DiegoCardoso) [reply vaadin-charts-flow-parent/vaadin-charts-flow-svg-generator/src/main/java/com/vaadin/flow/component/charts/export/NodeRunner.java:10]: That's going to be mentioned on the documentation, yes.

Comment (knoobie) [reply vaadin-charts-flow-parent/vaadin-charts-flow-svg-generator/src/main/java/com/vaadin/flow/component/charts/export/NodeRunner.java:10]: Having a note inside `SVGGenerators` javadoc wouldn't hurt I think

File: vaadin-charts-flow-parent/vaadin-charts-flow-svg-generator/src/main/java/com/vaadin/flow/component/charts/export/ExportOptions.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+    public void setTimeline(Boolean timeline) {
+        this.timeline = timeline;
+    }
+
+    public boolean getExecuteFunctions() {
+        return executeFunctions;
+    }
+
+    public void setExecuteFunctions(boolean executeFunctions) {
```
Comment (alvarezguille) [inline vaadin-charts-flow-parent/vaadin-charts-flow-svg-generator/src/main/java/com/vaadin/flow/component/charts/export/ExportOptions.java:72]:
```suggestion
    public void setExecuteFunctions(Boolean executeFunctions) {
```

**Review by alvarezguille — APPROVED** [review]

**Pull request merged**